### PR TITLE
Destination is not mandatory anymore in store_artifacts step

### PR DESCRIPTION
**Pull request by kimh**

### CODE REVIEW

File: jekyll/_cci2/configuration-reference.md [hunk 1]
```diff
@@ -503,7 +503,7 @@ Step to store artifacts (for example logs, binaries, etc) to be available in the
 Key | Required | Type | Description
 ----|-----------|------|------------
 path | Y | String | Directory in the primary container to save as build artifacts
-destination | Y | Prefix added to the artifact paths in the artifacts API
+destination | N | Prefix added to the artifact paths in the artifacts API
```
Comment (smaant) [inline jekyll/_cci2/configuration-reference.md]: Could you please also describe default value in case of not specified?

Comment (ndintenfass) [inline jekyll/_cci2/configuration-reference.md]: Near as I can tell, the behavior is to put the artifacts in the same path as the working directory?

Comment (smaant) [inline jekyll/_cci2/configuration-reference.md]: Not working directory, but the `path`

**Comment by kimh** [comment]
I'm working on something else now and will come back to take care of feedbacks here

**Comment by kimh** [comment]
Feedbacks were addressed. Merging.

**Pull request merged**